scripts/generate-clients/copy-to-clients.js

```diff
@@ -35,6 +35,11 @@ const mergeManifest = (fromContent = {}, toContent = {}) => {
   const merged = {};
   for (const name of Object.keys(fromContent)) {
     if (fromContent[name].constructor.name === "Object") {
+      if (name === "devDependencies") {
+        // Remove devDeps defined in monorepo root
+        const devDepsInRoot = ["downlevel-dts", "rimraf", "typedoc", "typescript"];
```

Comment:
Have you checked if we can also remove the `@aws-sdk/service-client-documentation-generator`?

Reply:
We should be able to remove `@aws-sdk/service-client-documentation-generator` if it's defined in root package.json
We can take it up in a separate PR as it's an internal dependency and will need additional testing with docs generation.